src/pages/play.astro

```diff
@@ -63,7 +63,7 @@ const t = useTranslations(lang)
   }
   .link-events {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(30rem, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(16.5rem, 1fr));
```

Comment:
This might be too small. We want some extra width for each item, so we can comfortably read the logs, and also not overflow on wallet address.

<img width="1275" height="669" alt="Image" src="https://github.com/user-attachments/assets/c0d389bd-3256-4128-a02c-667b8692f12e" />
At this particular screen size (13" Apple M1 Air), I'd expect to see only 2 items, not 4 in single row.